dlpy/model.py

```diff
@@ -271,11 +273,17 @@ def fit(self, data, inputs=None, target=None, data_specs=None, mini_batch_size=1
         # check whether the field is none or not
         if self.model_weights is not None and self.model_weights.to_table_params()['name'].upper() in \
                 list(self._retrieve_('table.tableinfo').TableInfo.Name):
-            print('NOTE: Training based on existing weights.')
-            init_weights = self.model_weights
+            if train_from_scratch:
+                print('NOTE: Ignoring the existing weights and training from scratch.')
+                init_weights = self.model_weights
```

Comment:
don't we initialize init_weights with model_weights in this way. so it still uses the trained weights table if there is any.